libnethack/dat/gehennom.des

```diff
@@ -205,11 +206,11 @@ RANDOM_PLACES:(04,02),(46,02),(04,15),(46,15)
 REGION:(00,00,50,17),unlit,"swamp"
 MAZEWALK:(00,09),west
 MAZEWALK:(50,08),east
-STAIR:levregion(01,00,11,20),(0,0,50,17),down
-STAIR:levregion(69,00,79,20),(0,0,50,17),up
-BRANCH:levregion(01,00,11,20),(0,0,50,17)
-TELEPORT_REGION:levregion(01,00,11,20),(0,0,50,17),up
-TELEPORT_REGION:levregion(69,00,79,20),(0,0,50,17),down
+STAIR:levregion(00,00,10,20),(0,0,50,17),down
```

Comment:
I had to tweak these numbers (it was complaining about being out of range). Please double check that the levels are sane with these changes from here to the rest of the file.